dynamic_publisher.py

```diff
@@ -23,18 +25,38 @@ class DynamicPublisher(PubSubConnectivity, TerminatorBlock):
     version = VersionProperty("0.1.0")
     topic = StringProperty(
         title="Topic",
-        default="")
+        default="",
+        order=0)
 
     ttl = TimeDeltaProperty(
         title="Time-to-live",
         advanced=True,
         order=0,
         default=dict(seconds=600))
+    is_local = BoolProperty(
+        advanced=True,
+        allow_none=True,
```

Comment:
Does this need to allow `None`? It's a bool, after all, and older configs without this property defined will use the default. Am I misunderstanding the intent?

Reply:
>            Exception: If a value has not been set and a default is not defined
            and allow_none is False.

Ahh, I didn't realize that allow_none didn't really do anything if a default had been set. Removing!